### PR TITLE
Suggestion property

### PR DESCRIPTION
**Pull request by irenelagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix [#51](https://github.com/hlxsites/hsf-commonmoves/issues/51)

Test URLs:
Before: https://main--hsf-commonmoves--hlxsites.hlx.page/search
-After: https://suggestion-property--hsf-commonmoves--hlxsites.hlx.page/search

### CODE REVIEW

File: blocks/property-search-bar/property-search-bar.css [hunk 1]
```diff
@@ -629,7 +684,7 @@ main .section .property-search-bar .btn.btn-secondary:hover {
     display: none;
 }
 
-.property-search-bar.block .no-scroll {
+.no-scroll {
```
Comment (bstopp) [inline blocks/property-search-bar/property-search-bar.css:687]: This shouldn't be in the block level CSS, or it needs to be more qualified.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: blocks/property-search-bar/property-search-bar.css [hunk 2]
```diff
@@ -911,7 +970,8 @@ main .section .property-search-bar .btn.btn-secondary:hover {
 
     .property-search-bar.block .search-listing-container > div:last-of-type {
         display: flex;
-        width: var(--normal-page-width);
+        width: 100vw;
```
[the remark this answers is not in the document]
Comment (irenelagno) [reply blocks/property-search-bar/property-search-bar.css:973]: yep. i need this to be full page width see [original website](https://www.commonmoves.com/search?SearchType=Map&PropertyType=1%2C2&ApplicationType=FOR_SALE&ListingStatus=1&NewListing=false&Sort=PRICE_DESCENDING&PageSize=32&Page=1&NorthEastLatitude=42.928308834863444&NorthEastLongitude=-68.11380499917098&SouthWestLatitude=41.017246455973805&SouthWestLongitude=-71.79422492104598&SearchParameter=%7B%22type%22%3A%22FeatureCollection%22%2C%22features%22%3A%5B%7B%22type%22%3A%22Feature%22%2C%22properties%22%3A%7B%7D%2C%22geometry%22%3A%7B%22type%22%3A%22Polygon%22%2C%22coordinates%22%3A%5B%5B%5B-71.79422492104598%2C42.928308834863444%5D%2C%5B-71.79422492104598%2C41.017246455973805%5D%2C%5B-68.11380499917098%2C41.017246455973805%5D%2C%5B-68.11380499917098%2C42.928308834863444%5D%2C%5B-71.79422492104598%2C42.928308834863444%5D%5D%5D%7D%7D%5D%7D&MapSearchType=MapBounds&isAgentPage=false&isFranchisePage=true&franchiseeCode=MA312)

File: blocks/property-search-bar/property-search-bar.css (hunk 1 above)
Comment (irenelagno) [reply blocks/property-search-bar/property-search-bar.css:687]: updated

Comment (bstopp) [reply blocks/property-search-bar/property-search-bar.css]: Ah i see now.

**Pull request merged**